docs/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+Write a description of the changes proposed in this pull request. Include images/GIFs if relevant for reviewers. Try Fireshot (Chrome, Firefox) for full-page screenshots and LICEcap (macOS) for GIFs.
```

Comment:
Firefox actually has built-in full page screenshots!
![image](https://user-images.githubusercontent.com/6129479/110679981-14e46a00-81a6-11eb-98cc-c06c5776b18c.png)
![image](https://user-images.githubusercontent.com/6129479/110680035-2299ef80-81a6-11eb-8ec3-2b59d82c66e0.png)

Reply:
oh cool, i'll recommend that instead. also i'll use the html comments feature i learned from you...